src/Microsoft.Azure.SignalR.Common/Auth/AuthUtility.cs

```diff
@@ -18,7 +18,7 @@ internal static class AuthUtility
 
         private static readonly JwtSecurityTokenHandlerSignalR JwtTokenHandler = new JwtSecurityTokenHandlerSignalR();
 
-        public static string GenerateJwtBearer(
```

Comment:
since the class is internal, no need to change the method to public

Reply:
fixed. 
1. change all public class to internal
2. keep the accessibility of all classes' methods and variables unchanged